example/main.go

```diff
@@ -54,18 +54,39 @@ func autoCompleteCallback(t *terminal.Terminal, line string, pos int, key rune)
 	return
 }
 
+func AddOrReplaceHistory(t *terminal.Terminal, replace bool, l string) {
+	// if in default auto mode, we don't manage history
+	// we also don't add empty commands. (at start of program)
+	if t.AutoHistory() || l == "" {
+		return
+	}
+	log.LogVf("Adding to history %q replace %t", l, replace)
+	if replace {
+		t.ReplaceLatest(l)
+	} else {
+		t.AddToHistory(l)
+	}
+}
+
 func Main() int {
 	// Pending https://github.com/golang/go/issues/68780
 	flagHistory := flag.String("history", "/tmp/terminal_history", "History `file` to use")
```

Comment:
I would explain quickly what is pending, because it requires extra lookup by opening the browser

Reply:
I can remove that now, as we have a semi permanent fork

Reply:
https://github.com/fortio/terminal/pull/10